ddtrace/tracer.py

```diff
@@ -840,6 +841,40 @@ def shutdown(self, timeout=None):
             pass
         atexit.unregister(self._atexit)
 
+    def get_correlation_ids(self, tracer=None):
```

Comment:
I don't think we need `tracer=`? seems weird given that this is now a method? wouldn't you just call `get_correlation_ids` on that instance instead of passing it in?

(would make more sense if this was a classmethod, not that I am suggesting we do that)

Reply:
Yup, but from the comments left over in this helper it seems like they wanted to account for OpenTracing tracers which I don't think have the same `get_correlation_ids()` instance method... I agree it would make more sense to keep as a class method but also kind of a weird spot 😕